src/algebra/big_operators/basic.lean

```diff
@@ -1491,6 +1491,21 @@ lemma prod_finset_coe [comm_monoid β] :
   ∏ (i : (s : set α)), f i = ∏ i in s, f i :=
```

Comment:
We seem to have `prod_finset_coe` twice in this file, once in the `fintype` namespace and once in the `finset` namespace. Can you remove one?

Reply:
I moved everything to the `finset` namespace, since that is the earliest definition/usage.